src/resources.py

```diff
@@ -1,130 +0,0 @@
-# Copyright 2021 Canonical
-# See LICENSE file for licensing details.
-import logging
-import requests
-import yaml
-from kubernetes import kubernetes
-from kubernetes import client, config, utils
-from kubernetes.utils import create_from_dict
-
-
-logger = logging.getLogger(__name__)
-
-
-class PortainerResources:
-    """Class to handle the creation and deletion of those Kubernetes resources
-    required by the Portainer UI, but not automatically handled by Juju"""
-
-    def __init__(self, charm):
-        self.model = charm.model
-        self.app = charm.app
-        self.config = charm.config
-        self.namespace = charm.namespace
-        self.charm_dir = charm.charm_dir
-        # Setup some Kubernetes API clients we'll need
-        kcl = kubernetes.client.ApiClient()
-        self.apps_api = kubernetes.client.AppsV1Api(kcl)
-        self.core_api = kubernetes.client.CoreV1Api(kcl)
-        self.auth_api = kubernetes.client.RbacAuthorizationV1Api(kcl)
-
-    def apply(self) -> None:
-        url='https://raw.githubusercontent.com/portainer/k8s/master/deploy/manifests/portainer/portainer.yaml'
-        resp = requests.get(url)
-        logger.info("Creating additional Kubernetes resources")
-        k8s_client=client.api_client.ApiClient(configuration=config.load_incluster_config())
-        ignore_resources = ["Namespace","ServiceAccount","ClusterRoleBinding","PersistentVolumeClaim","Deployment","Service"]
```

Comment:
This line filtered out *all* of the entries in the file you downloaded, making this whole method (and by extension, class) essentially a no-op